pybaseball/datahelpers/postprocessing.py

```diff
@@ -54,21 +81,30 @@ def try_parse(
     # Is it an float or an int (including percetages)?
     try:
         percentage = (value.endswith('%') or column_name.endswith('%') or column_name in known_percentages)
+        if percentage:
+            return try_parse_percentage(value)
         if '.' in value:
-            return float(value.strip(' %')) / (1 if not percentage else 100.0)
+            return float(value)
         else:
-            result = int(value.strip(' %'))
-            return result if not percentage else result / 100.0
+            return int(value)
     except:
         pass
 
     return value
 
 
-def coalesce_nulls(data: pd.DataFrame, value: Union[str, int, float] = np.nan) -> pd.DataFrame:
+def try_parse_percentage(value: str) -> float:
+    if '.' in value:
+        return float(value.strip(' %')) / 100.0
+    else:
+        result = int(value.strip(' %'))
```

Comment:
I think if you cast as int and then divide by 100.0 you'll change it to float, so that this is exactly the same as `float(value.strip(' %')) / 100.0` from above.